programs/client/Client.cpp

```diff
@@ -469,7 +470,9 @@ class Client : public Poco::Util::Application
         ///   The value of the option is used as the text of query (or of multiple queries).
         ///   If stdin is not a terminal, INSERT data for the first query is read from it.
         /// - stdin is not a terminal. In this case queries are read from it.
-        if (!stdin_is_a_tty || config().has("query"))
+        /// - -qf (--queries-file) command line option is present.
+        ///   The value of the option is used as file with query (or of multiple queries) to execute.
+        if (!stdin_is_a_tty || config().has("query") || config().has("queries-file"))
```

Comment:
@alexey-milovidov should we handle case and throw error if user specify both 'query' and 'queries-file' ?

Reply:
And what is the other option? All alternatives i can imagine sound counter-intuitive